ui/puzzle/src/interfaces.ts

```diff
@@ -52,9 +54,11 @@ export interface Controller extends KeyboardController {
   getNode(): Tree.Node;
   showComputer(): boolean;
   trans: Trans;
-  getData(): PuzzleData;
+  data: PuzzleData;
```

Comment:
I'm hoping that this change from getData to data is acceptable, since it further aligns the `RoundController` with `PuzzleController`

Reply:
@ornicar now that we don't need to change `getData()` to `data`, any preference for one over the other?

Reply:
I don't think just `data` works. `getData()` is here to provide the latest data, even after it's been replaced here:
https://github.com/brollin/lila/blob/2a35b11332f02640ce7db45bae8c5eed865b6619/ui/puzzle/src/ctrl.ts#L101-L102

If you don't use `getData()`, then `ctrl.data` will keep on pointing to the old data.